container-search/src/main/java/com/yahoo/search/query/profile/QueryProfileCompiler.java

```diff
@@ -38,13 +35,15 @@ public static CompiledQueryProfile compile(QueryProfile in, CompiledQueryProfile
             DimensionalMap.Builder<CompoundName, QueryProfileType> types = new DimensionalMap.Builder<>();
             DimensionalMap.Builder<CompoundName, Object> references = new DimensionalMap.Builder<>();
             DimensionalMap.Builder<CompoundName, Object> unoverridables = new DimensionalMap.Builder<>();
+            System.out.println("Compiling " + in.toString());
```

Comment:
I do not think you want these debug statements lying around.

Reply:
Ops, thanks